tests/sentry/search/events/test_builder.py

```diff
@@ -24,8 +24,12 @@
 
 class QueryBuilderTest(TestCase):
     def setUp(self):
-        self.start = datetime.datetime(2015, 5, 18, 10, 15, 1, tzinfo=timezone.utc)
-        self.end = datetime.datetime(2015, 5, 19, 10, 15, 1, tzinfo=timezone.utc)
+        self.start = datetime.datetime.now(tz=timezone.utc).replace(
+            hour=10, minute=15, second=0, microsecond=0
+        ) - datetime.timedelta(days=1)
+        self.end = datetime.datetime.now(tz=timezone.utc).replace(
```

Comment:
might need to do days=2 for the timedelta otherwise if a test runs before 10:15 things might be funny

Reply:
Would it make sense to do the replace after the timedelta then?